R/columnchart.R

```diff
@@ -846,10 +866,57 @@ Column <- function(x,
                     xaxis = if (NCOL(chart.matrix) > 1) "x2" else "x", yaxis = "y",
                     data.label.font[[i]], is.stacked, data.label.centered)
 
+        # Create annotations separately for each series
+        # so they can be toggled using the legend 
+        for (j in seq_along(overlay.annotation.list))
+        {
+            a.tmp <- overlay.annotation.list[[j]]
```

Comment:
Could update the code to perhaps `for (current.annot in overlay.annotation.list)` to avoid the generic ambiguous name of `a.tmp`. Makes it easier to understand the code is doing annotation manipulation/processing. Perhaps similar for the resulting variable names of `tmp.text` to `annot.text` etc.

Reply:
In 7911b18, I've renamed 
`j` to `curr.annot.ind`
`a.tmp` to `curr.annot`
`tmp.text` to `curr.annot.text`

Reply:
Looks good. I probably should've been more explicit but for completeness there are still some `tmp.*` variable names with `tmp.ypos` `tmp.dat` and `tmp.align`. These could be updated too?